prelude/prelude.purs

```diff
@@ -159,9 +192,34 @@ module Prelude
 
   infixl 4 <*>
 
+  -- | Applys are intuitively Applicatives less `pure`, and more formally a strong lax semi-
```

Comment:
This could possibly put some beginners off, while being technically correct (I _think_ :smile:)

The way I explain this in the book is that `Apply` allows us to lift functions of `n > 1` arguments over a the type constructor `f`, i.e. to take functions `a -> b -> ...` to functions `f a -> f b -> f c`. `Functor` and `Applicative` then give the `n=0` and `n=1` cases respectively.

Not sure what's best, this is just my 2¢.